src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 10; i++) {
```

Comment:
You should create three balls using the getRandomBall() method from the Lottery class. 

Also, 10 is magic number, create constant